app/models/dog_searcher.rb

```diff
@@ -31,25 +33,24 @@ def initialize(params: {}, manager: false)
   end
 
   def search
-    @dogs = Dog.includes(:photos, :foster)
-    if @manager
-      @dogs = @dogs.includes(:adopters, :comments)
+    if @manager # manager view
+      @dogs = Dog.includes(:adoptions, :adopters, :comments)
```

Comment:
were `:photos` and `:foster` not doing anything?

Reply:
same with removing `with_includes`, so `:primary_breed` and removing photos from there is ok since it was a dupe, oops